types/lnrpc.d.ts

```diff
@@ -1,8 +1,11 @@
 import { Duplex, Readable } from "./streams";
+import { ChanBackupSnapshot, ChannelPoint, EstimateFeeResponse, EstimateFeeRequest, ExportChannelBackupRequest, ChanBackupExportRequest, VerifyChanBackupResponse, RestoreBackupResponse, ChannelBackupSubscription } from "./generated/rpc_pb";
```

Comment:
Don't import from the generated types. Re-declare in this file. The developer experience of digging down into the generated types to view properties is poor.

Reply:
that was autogenerated but will fix.